docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: Continuous Integration
+parent: Getting started
+nav_order: 5
+permalink: /getting-started/continuous-integration/
+---
+
+# Continuous Integration
+
+OSS-Fuzz offers Continuous Integration(CI) support for projects hosted on
+GitHub. This service is called CIFuzz and can be used to run OSS-Fuzz fuzz targets
+on your project when a pull request is submitted. It has the advantage of
+detecting bugs before they are committed into your project. This insures that
+less bugs are committed meaning less active security vulnerabilities.
+
+## How it works
+
+CIFuzz works by checking out a repository at the head of a pull request. The
+project's fuzz targets are built and run for
+a definite amount of time (default is 10 minutes). If a bug is found, the
```

Comment:
This seems to be a bit misleading. If all the fuzz targets were run in parallel for 10 minutes, I'd turn CIFuzz on right away but it appears in the foreseeable future it's going to run them one by one for (600 / the number of fuzz targets) seconds. Would it be possible to mention that to make it easier to decided whether it would make sense to turn it on without reading the code of the action? It would probably be also helpful if it was mentioned that fuzz targets are built with ASan only and run with no corpora (or, more precisely, without the corpora OSSFuzz has been accumulating).

Reply:
OK. @Leo-Neat Can you change this to mention ASAN only and that each fuzz target that if N fuzz targets are run, each will get run (unless there is a bug?) for total_time/N seconds.
@evverx 
I don't think we are going to mention corpus or every fuzzer being run so explicitly since in the coming weeks we should have features that would allow some use of corpus and avoiding the need to run every fuzzer.